apps/api/cli.README.md

```diff
@@ -0,0 +1,60 @@
+# TODO Break `coscrad-cli` out into a separate app
+
+# COSCRAD CLI
+
+## About
+
+`coscrad-cli` is a command line tool for internal use for data management.
```

Comment:
For data management at the admin level (or something like that)

Reply:
Ok. Reworded.